src/Bolero.Server/Bolero.Server.fsproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
Unfortunately this is a breaking change.

There was a change in ASP.NET Core 7, namely a method added in an interface, which we had to take into account here: https://github.com/fsbolero/Bolero/blob/1eb9ee3f3ce11968f6052f1a82d355f4a3d1ef33/src/Bolero.Server/Remoting/Endpoints.fs#L153-L155

So if we remove net7.0 as a target, then .NET 7 applications will fall back on the net6.0 version of Bolero.Server, which is missing this part.

Reply:
OK, I’ll fix